WalletWasabi.Fluent/Views/Wallets/WalletSettingsView.axaml

```diff
@@ -27,7 +27,7 @@
         <c:InfoMessage>
           <DockPanel>
             <Button Margin="50 0" Command="{Binding VerifyRecoveryWordsCommand}" Content="Verify Recovery Words" DockPanel.Dock="Right" />
-            <TextBlock Text="It is recommended that you verify your wallet Recovery Words from time to time." />
+            <TextBlock Text="It is recommended that you verify your wallet Recovery Words from time to time. Your password is also required to restore your backup, do not forget it." />
```

Comment:
I don't necessarily like how the symmetry is broken by adding an extra ! since there's no button on the other side to accompany it, but I agree with the extra emphasis since the password is just as important as the 12 words.

Reply:
I'd use the yellow styling that we use in when a config requires relaunch. I think it would be visually the best.